src/components/TheFooter.vue

```diff
@@ -5,18 +5,14 @@
   </p>
 </template>
 
-<script>
-export default {
-  name: 'TheFooter',
-  data() {
-    return {
-      year: ''
-    }
-  },
-  mounted() {
-    this.year = new Date().getFullYear()
-  }
-}
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+
+const year = ref(0)
+
+onMounted(() => {
+  year.value = new Date().getFullYear()
+})
```

Comment:
onMounted is not needed in this case. Just set the value inside the `ref`

Reply:
Yeah, I actually realized this yesterday, after pushing and asking this question 🙈
Sorry for being stupid, **corrected**.